src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -189,7 +189,17 @@ async def get(self):
         permitted_gardens = self.permissioned_queryset(Garden, GARDEN_READ)
         response_gardens = []
 
-        for garden in permitted_gardens.no_cache():
+        permitted_gardens_list = list(
+            permitted_gardens.filter(connection_type__ne="LOCAL").no_cache()
+        )
+        _local_garden = local_garden()
```

Comment:
I only thought about this while testing it, but it turns out that for remote gardens that themselves have remote gardens (aka a "child" that has "grandchildren"), we list all known systems, not just the local ones.  I'm not particularly fond of that behavior, but the right thing to do here is probably to replicate it with the local garden.  You can do that by instead calling:

```python
_local_garden = local_garden(all_systems=True)
```

That should result in the response including not just the systems of the local garden, but all remote gardens as well.